### DISCUSSION TITLE
GBA/GC Link, with - GC BlueRetro Wireless and 8BitDo Gbros., or equivalent?

### DISCUSSION THREAD

**Discussion by deadbolt55** [darthcloud/BlueRetro#1029] [General]
Hello all,

I was curious if anyone knows, or has tried linking a GBA to the GameCube via this combination wirelessly, instead of using the original, Official link cable for GBA-to-GC?

I'm curious because it would be quite handy for modern setups, avoiding to have a cable extended across the room to make use of the GBA functions within certain GC games, along with the GBA connected on another port using the BlueRetro wireless adapter instead.

Thank you all for your time,

-Carson

**Comment by darthcloud** [comment]
That won't work, Gbros only support regular wired GC controller

**Reply by Tredes** [reply]
Hi darthcloud

I understand the project is at your discretion and availability, and I just want to contribute with a few questions: the BlueRetro is purely a receiver; if it's to be used with the Gameboy Advance and possibly replace the "link cable," an ESP32 module would be needed to act as a transmitter, connected to the internal board on the cable or directly to the GBA device's board. The handheld console is essential, but based on this idea, it would be possible to use the Gameboy Advance and Gameboy Micro. I hope you can evaluate it or share it with a colleague. Thanks for your attention.

(web page for functional reference:  https://retrogames.cl/foro/viewtopic.php?t=11995 )
